lib/main.js

```diff
@@ -70,14 +71,22 @@ export default {
     this.subscriptions.add(atom.commands.add('atom-text-editor', {
       'linter-eslint:fix-file': function() {
         const textEditor = atom.workspace.getActiveTextEditor()
+        const filePath = textEditor.getPath()
+        const fileDir = Path.dirname(filePath)
+        const params = {
+          fileDir,
+          global: atom.config.get('linter-eslint.useGlobalEslint'),
+          nodePath: atom.config.get('linter-eslint.globalNodePath'),
+        }
+        const eslintDir = findEslintDir(params)
```

Comment:
[Here](https://github.com/AtomLinter/linter-eslint/blob/master/lib/main.js#L125-L138)'s how you can give a job to worker, the `request` method will return a promise that will resolve when worker replies, You can either return a promise or a value from the worker, like it's done [here](https://github.com/AtomLinter/linter-eslint/blob/master/lib/worker.js#L32), [here](https://github.com/AtomLinter/linter-eslint/blob/master/lib/worker.js#L39) and [here](https://github.com/AtomLinter/linter-eslint/blob/master/lib/worker.js#L88)